Sources/Rendering/Core/Picker/test/testPicker.js

```diff
@@ -0,0 +1,146 @@
+import test from 'tape';
+import testUtils from 'vtk.js/Sources/Testing/testUtils';
+
+import 'vtk.js/Sources/Rendering/Misc/RenderingAPIs';
+import vtkRenderWindow from 'vtk.js/Sources/Rendering/Core/RenderWindow';
+import vtkRenderer from 'vtk.js/Sources/Rendering/Core/Renderer';
+import vtkPicker from 'vtk.js/Sources/Rendering/Core/Picker';
+import vtkActor from 'vtk.js/Sources/Rendering/Core/Actor';
+import vtkConeSource from 'vtk.js/Sources/Filters/Sources/ConeSource';
+import vtkMapper from 'vtk.js/Sources/Rendering/Core/Mapper';
+
+function setupRenderingComponents(gc) {
+  const container = document.querySelector('body');
+  const renderWindowContainer = gc.registerDOMElement(
+    document.createElement('div')
+  );
+  container.appendChild(renderWindowContainer);
+
+  // create what we will view
+  const renderWindow = gc.registerResource(vtkRenderWindow.newInstance());
+  const renderer = gc.registerResource(vtkRenderer.newInstance());
+  renderWindow.addRenderer(renderer);
+
+  // now create something to view the scene, in this case webgl
+  const glwindow = gc.registerResource(renderWindow.newAPISpecificView());
+  glwindow.setContainer(renderWindowContainer);
+  renderWindow.addView(glwindow);
+  glwindow.setSize(400, 400);
+
+  renderWindow.render();
+
+  return renderer;
+}
+
+function createRenderingPipeline(filter) {
+  const mapper = vtkMapper.newInstance();
+  mapper.setInputConnection(filter.getOutputPort());
+  const actor = vtkActor.newInstance();
+  actor.setMapper(mapper);
+
+  return actor;
+}
+
+test.onlyIfWebGL('vtkPicker.pick3DPoint - cone picking', (t) => {
+  const gc = testUtils.createGarbageCollector(t);
+  const renderer = setupRenderingComponents(gc);
+
+  const coneSource = vtkConeSource.newInstance({
+    center: [0, 0, 0],
+    radius: 1,
+    height: 2,
+    direction: [1, 0, 0],
+  });
+
+  const coneActor = createRenderingPipeline(coneSource);
+  renderer.addActor(coneActor);
+
+  const picker = vtkPicker.newInstance();
+
+  const selectionPoint = [-1, 0, 0, 1.0];
+  const focalPoint = [2, 0, 0, 1.0];
+  picker.pick3DPoint(selectionPoint, focalPoint, renderer);
+
+  const pickedActors = picker.getActors();
+  t.equal(pickedActors[0], coneActor);
+
+  gc.releaseResources();
```

Comment:
no `t.end()`?

Reply:
`gc.releaseResources()` calls `t.end()`. Not obvious from the call, but that's what the function does. The context comes from `testUtils.createGarbageCollector(t)` above.